### PR TITLE
APP-2665 Fix: Create DAO Wizard - Step 4 Missing Link

### PR DESCRIPTION
**Pull request by thekidnamedkd**
#### Issue before:
image

missing link out for {{this guide}}

#### After fix:
Pasted Graphic 16

This version uses the step 4 key values which would correspond with the JSON notation pattern of the previous 3 Steps with createDAO.step4.{...} (note camel case + upper).

#### Optionally (see `/pages/createDao.tsx` code comments + comment below): 
Without knowing the spec of choice, I also updated the previous [common.json](https://github.com/aragon/app/blob/develop/src/locales/en/common.json) key values for original createDao.stepCommunityVoting.{...} (note camel case + lower) to include `descLinkLabel` and `descLinkURL` which fixes the template output while maintaining the current production copy:

Pasted Graphic 17

`<Step />` props to change output are as follows (also in code comments), respectively:

```
wizardTitle={t('createDAO.step4.title')}
wizardDescription={htmlIn(t)('createDAO.step4.description')}
```

or 

```
wizardTitle={t('createDao.stepCommunityVoting.title')}
wizardDescription={htmlIn(t)('createDao.stepCommunityVoting.desc')}
```

Task: N/A (unknown ticket)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I ran all tests with success and extended them if necessary.

### CODE REVIEW

File: src/pages/createDAO.tsx [hunk 1]
```diff
@@ -391,8 +391,11 @@ export const CreateDAO: React.FC = () => {
             <SetupCommunity />
           </Step>
           <Step
-            wizardTitle={t('createDao.stepCommunityVoting.title')}
-            wizardDescription={htmlIn(t)('createDao.stepCommunityVoting.desc')}
+            // *** optionally leaving previous common.js pointers which now work correctly also ***
+            // wizardTitle={t('createDao.stepCommunityVoting.title')}
+            // wizardDescription={htmlIn(t)('createDao.stepCommunityVoting.desc')}
+            wizardTitle={t('createDAO.step4.title')}
+            wizardDescription={htmlIn(t)('createDAO.step4.description')}
```
Comment (thekidnamedkd) [inline src/pages/createDAO.tsx:398]: See here regarding Step prop options.

**Comment by Barukimang** [comment]
Hej @thekidnamedkd ! Great job on spotting this issue, we hadn't seen it yet!

Your fix looks good. For me to double check it locally and to be able to merge it, the fix should be done directly on the App repo instead of on a fork (permissions issue).

Our naming conventions for fix branches are `f/<one-or-2-word-description-of-fix>`

**Comment by evanaronson** [comment]
Thanks @thekidnamedkd.

APP-2665 for tracking in Jira.

**Comment by thekidnamedkd** [comment]
> Hej @thekidnamedkd ! Great job on spotting this issue, we hadn't seen it yet!
> 
> Your fix looks good. For me to double check it locally and to be able to merge it, the fix should be done directly on the App repo instead of on a fork (permissions issue).
> 
> Our naming conventions for fix branches are `f/<one-or-2-word-description-of-fix>`

Hey thanks! I tried a direct SSH clone of this repo and push with the naming convention on the branch -- `f/2665`. Getting a 403 in the terminal as a non collaborator I suspect. Anything I need to do regarding permissions to direct-in?

**Comment by Barukimang** [comment]
@thekidnamedkd thanks for trying this. Yes it's a permission issue. Will ask someone from our team. We're all on shut down over holiday period now, so response can take some time.

Hej @RakeshUP do you know how we can accept incoming PRs from community contributors? See ☝️ Do you have the rights to make this possible?

**Review by RakeshUP — APPROVED** [review]
The code changes look good! I'm approving it to check if it's possible to merge from a fork. Even if it's blocked, James should be able to merge! @Barukimang 

@thekidnamedkd, we have another pre-requisite. The code commits need to be signed like mentioned here https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

`git rebase --exec 'git commit --amend --no-edit -n -S' -i HEAD~2` to amend your commit to include a signature once you've configured your GPG keys. `HEAD~2` in the command being the number of commits, in your case 2.

**Comment by thekidnamedkd** [comment]
@RakeshUP thanks for the snippet and info! I've made the changes to sign both commits with GPG and updated the branch on the fork to the correct naming convention. Would you like a fresh PR for clarity? I believe it would be required to make use of the branch naming change.

**Comment by RakeshUP** [comment]
@thekidnamedkd It's fine, you don't need to create a new PR this time. I'll do a quick test locally and merge this PR!

**Comment by Barukimang** [comment]
The fix is live now! Thanks for this one @thekidnamedkd, and @RakeshUP for following this up.

Image
image

**Pull request merged**